#!/usr/bin/env python
PACKAGE = "simple_follower"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("HSV_H_MAX", int_t, 0, "HSV_H_MAX", 0, 0, 255)
gen.add("HSV_H_MIN", int_t, 0, "HSV_H_MIN", 192, 0, 255)
gen.add("HSV_S_MAX", int_t, 0, "HSV_S_MAX", 194, 0, 255)
gen.add("HSV_S_MIN", int_t, 0, "HSV_S_MIN", 251, 0, 255)
gen.add("HSV_V_MAX", int_t, 0, "HSV_V_MAX", 104, 0, 255)
gen.add("HSV_V_MIN", int_t, 0, "HSV_V_MIN", 157, 0, 255)

exit(gen.generate(PACKAGE, "simple_follower", "HSV_Parament"))

